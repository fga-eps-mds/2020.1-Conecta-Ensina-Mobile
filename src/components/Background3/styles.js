import styled from 'styled-components/native';

export const BlueContainer = styled.View`
  background-color: ${(props) => props.theme.colors.fundoAzul};
  align-items: center;
  flex: 1;
  width: 100%;
`;

export const WhiteContainer = styled.View`
  background-color: ${(props) => props.theme.colors.branco};
  align-items: center;
  height: 72%;
  width: 88.88%;
  border-radius: 20px;
  margin-top: 1.5%;
`;

export const Logo = styled.Image`
  width: 31.6%;
  height: 17.8%;
  margin-top: 1.25%;
`;
