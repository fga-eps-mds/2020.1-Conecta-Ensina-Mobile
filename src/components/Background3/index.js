import React from 'react';
import {BlueContainer, WhiteContainer, Logo} from './styles';
import Theme from '../../../Theme';

export default function Background3(props) {
  return (
    <Theme>
      <BlueContainer>
        <Logo source={require('../../assets/logo.png')} />
        <WhiteContainer>{props.children}</WhiteContainer>
      </BlueContainer>
    </Theme>
  );
}
