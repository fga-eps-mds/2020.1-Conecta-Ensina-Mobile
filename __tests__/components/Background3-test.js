import React from 'react';
import 'react-native';
import renderer from 'react-test-renderer';

import Background3 from '../../src/components/Background3';

it('renders correctly', () => {
  const tree = renderer.create(<Background3 />).toJSON();
  expect(tree).toMatchSnapshot();
});
