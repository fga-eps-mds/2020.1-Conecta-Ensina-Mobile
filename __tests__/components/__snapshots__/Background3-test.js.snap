// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#184E79",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "width": "100%",
      },
    ]
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../src/assets/logo.png",
      }
    }
    style={
      Array [
        Object {
          "height": "17.8%",
          "marginTop": "1.25%",
          "width": "31.6%",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "backgroundColor": "#ffffff",
          "borderRadius": 20,
          "height": "72%",
          "marginTop": "1.5%",
          "width": "88.88%",
        },
      ]
    }
  />
</View>
`;
